invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/components/RecordsResultsListItem.js

```diff
@@ -14,8 +14,27 @@ import PropTypes from "prop-types";
 import { Item, Label, Icon } from "semantic-ui-react";
 import { buildUID } from "react-searchkit";
 import { CompactStats } from "./CompactStats";
+import { Image } from "react-invenio-forms";
 
 class RecordsResultsListItem extends Component {
+  display_community = () => {
```

Comment:
minor: This is duplicated, it would be better to define it in common place, to avoid redundancy.
that accepts communities object.